src/components/detail/imageCarousel.tsx

```diff
@@ -26,24 +27,30 @@ function ImageCarousel() {
 
   return (
     <Slider {...settings}>
-      {imageURLs?.map((imageURL: string, index: number) => {
-        return (
-          <section key={index}>
-            <p className="relative h-48 w-full bg-indigo-500">
-              <Image
-                src={imageURL}
-                alt="place image"
-                fill
-                style={{
-                  background: ' #3687ff',
-                }}
-                placeholder="blur"
-                blurDataURL={`/_next/image?url=${imageURL}&w=16&q=1`}
-              />
-            </p>
-          </section>
-        );
-      })}
+      {imageURLs.length ? (
+        imageURLs?.map((imageURL: string, index: number) => {
+          return (
+            <section key={index}>
+              <p className="relative h-48 w-full">
+                <Image
+                  src={imageURL}
+                  alt="place image"
+                  fill
+                  style={{
+                    background: '#ECEFF5',
+                  }}
+                  placeholder="blur"
+                  blurDataURL={`/_next/image?url=${imageURL}&w=16&q=1`}
+                />
+              </p>
+            </section>
+          );
+        })
+      ) : (
+        <p className="relative h-48 w-full">
+          <Image alt="default place image" src={default_place} fill />
+        </p>
```

Comment:
이미지가 없으면 기본 이미지를 띄워줍니다.

Reply:
둘 다 Image 컴포넌트를 p 태그로 감싸고 있고, 스타일까지 동일하니까
재사용 가능한 컴포넌트 하나를 만드는 건 어떨까요?

```tsx
<p className="relative h-48 w-full">
    <Image alt="" src={imageURL 또는 default_place} fill />
</p>
```

🔼 요 형태가 똑같으니까 반복되는 코드를 재사용할 수 있게 만들면 좋을 것 같아요!
예를 들면 아래와 같은 형태로 수정할 수 있을 것 같아요.

```suggestion
      {imageURLs.length ? (
        imageURLs?.map((imageURL: string, index: number) => <PlaceImage props={필요한 props}/>)
      ) : (
        <PlaceImage props={}/>
        )
        
        또는
        const PlaceImageList = imageURLs?.map((imageURL: string, index: number) => <PlaceImage props={필요한 props}/>)
        {imageURLs.length ? (
        <PlaceImageList  />
      ) : (
        <PlaceImage props={}/>
        )
```

Reply:
오 그게 더 깔끔하네요! 반영해볼게요!! 고마워요~